core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -210,6 +223,15 @@ private CompletionSuggestionBuilder contexts(XContentBuilder contextBuilder) {
         return this;
     }
 
+    public boolean skipDuplicates() {
```

Comment:
nit: maybe add some javadoc since this is what most users see that use the java api